services/src/modules/plugins/index.ts

```diff
@@ -39,3 +40,23 @@ export async function buildArgumentInjectionGlobals(): Promise<Record<string, un
     .filter(p => p.addArgumentInjectionGlobals)
     .reduce(async (acc, cur) => ({ ...acc, ...(await cur.addArgumentInjectionGlobals!()) }), {});
 }
+
+export async function transformResourcesUpdates(
+  resourceGroup: Partial<ResourceGroup>
+): Promise<Partial<ResourceGroup>> {
+  return plugins
+    .filter(p => p.transformResourcesUpdates)
+    .reduce<Partial<ResourceGroup> | Promise<Partial<ResourceGroup>>>(
+      async (res, cur) => cur.transformResourcesUpdates!(await res),
+      resourceGroup
+    );
+}
+
+export async function transformResourceGroup(resourceGroup: ResourceGroup): Promise<ResourceGroup> {
```

Comment:
You didn't actually call this in `loadPlugins`

Reply:
It shouldn't be called in loadPlugins it's called in fetchAndValidate.